scripts/health.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+to_status() {
+  if [[ $1 == 0 ]]; then
+    printf "✅"
+    return
+  fi
+  printf "❌"
+}
+
+check_go_mod_tidy() {
+  if [[ ! -f go.mod || ! -f go.sum ]]; then
+    echo 1
+    return
+  fi
+  cp go.mod go.mod.orig
+  cp go.sum go.sum.orig
+  go mod tidy &> /dev/null
+  diff go.mod go.mod.orig > /dev/null
+  if [[ $? != 0 ]]; then
+    echo 1
+    return
+  fi
+  diff go.sum go.sum.orig > /dev/null
+  echo $?
+}
+
+check_go_fmt() {
+  out=$(gofmt -l . | xargs)
+  if [[ -z "$out" ]]; then
+    echo 0
+    return
+  fi
+  echo 1
+}
+
+for repo in $(node get-repos/index.js -l Go | jq -r '.[]'); do
+  tmp=$(mktemp -d)
+  pushd $tmp > /dev/null
+  git clone -q https://github.com/$repo .
```

Comment:
so... it _would_ be nice if there were a way to tell this to clone from local repos... E.g., from a gopath.

Maybe just make the `https://github.com` part configurable?